pom.xml

```diff
@@ -5,13 +5,13 @@
 	<parent>
 		<groupId>org.scijava</groupId>
 		<artifactId>pom-scijava</artifactId>
-		<version>17.1.1</version>
+		<version>24.2.0</version>
 		<relativePath />
 	</parent>
 
 	<groupId>net.imagej</groupId>
 	<artifactId>imagej-plugins-uploader-webdav</artifactId>
-	<version>0.2.3-SNAPSHOT</version>
+	<version>0.3.0-SNAPSHOT</version>
```

Comment:
Can we define properties for these and then use them as `<version>${imagej-plugins-uploader-webdav.version}</version>`, to allow them to be overridden by tooling such as the melting pot script?

Reply:
For the parent and artifact `<version>` values we do not (and should not) use properties. But for dependency version declarations, we should!